CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Version changelog
 
+## [Release] Release v0.239.0
+
+### New feature announcement
+
+You can now manage Databricks Apps using DABs by defining an `app` resource in your bundle configuration.
+For more information see Databricks documentation https://docs.databricks.com/en/dev-tools/bundles/resources.html#app
+
+CLI:
+ * Filter out system clusters in cluster picker ([#2131](https://github.com/databricks/cli/pull/2131)).
+ * Process all the fields in top level request object even if it contains request body ([#2155](https://github.com/databricks/cli/pull/2155)).
+
+Bundles:
+ * Added support for Databricks Apps in DABs ([#1928](https://github.com/databricks/cli/pull/1928)).
+ * Allow artifact path to be located outside the sync root ([#2128](https://github.com/databricks/cli/pull/2128)).
+ * Retry app deployment if there is an active deployment in progress ([#2153](https://github.com/databricks/cli/pull/2153)).
+ * Resolve variables in a loop ([#2164](https://github.com/databricks/cli/pull/2164)).
```

Comment:
Highlight and call out that nested complex variables are supported now?

Reply:
>  Resolve variables in a loop ([#2164](https://github.com/databricks/cli/pull/2164)).
> Improve resolution of complex variables within complex variables ([#2157](https://github.com/databricks/cli/pull/2157)).

Should be combined to "Fix resolution of complex variables #2164 #2157", the two PRs are part of the same work.